pyproject.toml

```diff
@@ -88,7 +87,7 @@ tls = [
 conch = [
     "cryptography >= 3.3",
     "appdirs >= 1.4.0",
-    "bcrypt >= 3.1.3",
+    "bcrypt >= 3.2.1",
```

Comment:
Just asking. Why do we need a newer version of bcrypt ?

Reply:
This is the oldest version of bcrypt that supports python3.9.